src/tests/ftest/cart/util/cart_logparse.py

```diff
@@ -10,10 +10,10 @@
 This provides a way of querying CaRT logfiles for processing.
 """
 
-from collections import OrderedDict
 import bz2
 import os
 import re
+from collections import OrderedDict
```

Comment:
I think dicts are ordered in python3 anyway so we could probably just remove this.

Reply:
For python 3.6, they are only guaranteed to be ordered in CPython. It's not until 3.7 that they are guaranteed to be ordered in all implementations

Reply:
But that's beyond scope here anyway